### PR TITLE
Source hubspot: add Feedback Submissions stream and update Ticket Pipelines stream

### PR DESCRIPTION
**Pull request by vladimir-remar**
## What
Add Feedback Submissions as new stream and update/use the Ticket Pipelines stream.

## How
In Feedback Submissions  just add an entry into `client.py` in the Ticket Pipelines since it was not use at all, I updated the class and add an entry into `client.py`.

## 🚨 User Impact 🚨
Not really, because `ticket_pipelines` as I said it wasn't use.

## Pre-merge Checklist
Expand the relevant checklist and delete the others. 

<details><summary> <strong> Updating a connector </strong></summary>
<p>
   
#### Community member or Airbyter
   
- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

### CODE REVIEW

**Comment by marcosmarxm** [comment]
Looks good @vladimir-remar! thanks for your contribution, please bump the connector version in config files, docker and docs? See instructions [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

**Comment by marcosmarxm** [comment]
Our integration account is returning 404 from the new endpoint. @vladimir-remar do you know how to give permission to v3?

**Comment by vladimir-remar** [comment]
> Our integration account is returning 404 from the new endpoint. @vladimir-remar do you know how to give permission to v3?

Hello @marcosmarxm, certainly not, I get the api-key by an external agent, let me ask.

**Comment by vladimir-remar** [comment]
Hello @marcosmarxm can you take at look on this?

**Comment by marcosmarxm** [comment]
@vladimir-remar is it possible to you share the output from `./gradlew airbyte-integrations:connectors:source-hubspot:integrationTest`?

File: airbyte-integrations/connectors/source-hubspot/integration_tests/abnormal_state.json [hunk 1]
```diff
@@ -14,6 +14,9 @@
   "email_events": {
     "timestamp": "2221-10-12T13:37:56.412000+00:00"
   },
+  "feedback_submissions": {
+    "timestamp": "2221-10-12T13:37:56.412000+00:00"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-hubspot/integration_tests/abnormal_state.json:18]:
```suggestion
    "updatedAt": "2221-10-12T13:37:56.412000+00:00"
```

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
small comments after is ready to merge! please share the output from integration tests

**Comment by vladimir-remar** [comment]
> @vladimir-remar is it possible to you share the output from `./gradlew airbyte-integrations:connectors:source-hubspot:integrationTest`?

Hi @marcosmarxm , I'm having problems with the tests, let me explain, I have 2 API-Keys, one for production and one without data, the problem is that if I use the prod API key the tests don't end because they reach the limit and end up giving a timeout, even with a recent `start_date`, on the other hand if I use the other API key they fail because there is certainly no data.
Attached files with errors.
[hubspot_tests_test_api_key.log](https://github.com/airbytehq/airbyte/files/7896669/hubspot_tests_test_api_key.log)
[huspot_tests_prod_api_key.log](https://github.com/airbytehq/airbyte/files/7896670/huspot_tests_prod_api_key.log)

**Review by marcosmarxm — APPROVED** [review]
thanks @vladimir-remar

**Pull request merged**